ner_v2/detectors/numeral/number_range/standard_number_range_detector.py

```diff
@@ -76,43 +72,48 @@ def init_regex_for_range(self, data_directory_path):
-                                       if(value.position == 1 and value.range_type == NUMBER_RANGE_MIN_TYPE)]
+        self.min_range_suffix_variants = [variant for variant, value in self.range_variants_map.items()
+                                          if (value.position == 1 and
+                                          value.range_type == numeral_constant.NUMBER_RANGE_MIN_TYPE)]
 
-        self.max_range_start_variants = [variant for variant, value in self.range_variants_map.items()
-                                         if (value.position == -1 and value.range_type == NUMBER_RANGE_MAX_TYPE)]
+        self.max_range_prefix_variants = [variant for variant, value in self.range_variants_map.items()
+                                          if (value.position == -1 and
+                                              value.range_type == numeral_constant.NUMBER_RANGE_MAX_TYPE)]
 
-        self.max_range_end_variants = [variant for variant, value in self.range_variants_map.items()
-                                       if (value.position == 1 and value.range_type == NUMBER_RANGE_MAX_TYPE)]
+        self.max_range_suffix_variants = [variant for variant, value in self.range_variants_map.items()
+                                          if (value.position == 1 and
+                                              value.range_type == numeral_constant.NUMBER_RANGE_MAX_TYPE)]
 
         self.min_max_range_variants = [variant for variant, value in self.range_variants_map.items()
-                                       if (value.position == 0 and value.range_type == NUMBER_RANGE_MIN_MAX_TYPE)]
+                                       if (value.position == 0 and
+                                           value.range_type == numeral_constant.NUMBER_RANGE_MIN_MAX_TYPE)]
 
     def _tag_number_in_text(self, processed_text):
         """
         replace number in text with number tag from number_detected_map
         Args:
             processed_text (str): processed text
         Returns:
-            tagged_number_text(str): text with number replaced with tag
+            (str): text with number replaced with tag
         Examples:
-            [In]  >>  text = 'i want to buy 3 apples and more than two bananas'
-            [In]  >>  number_detected_map = {'__number__1': ({'value': '3', 'unit': None}, '3'),
-                                             '__number__2': ({'value': '2', 'unit': None}, 'two')}
-            [In]  >> _tag_number_in_text(text)
-            [Out] >> 'i want to buy __number__1 apples and more than __number__2 bananas'
+            >>> text = 'i want to buy 3 apples and more than two bananas'
+            >>> number_detected_map = {'__number__1': ({'value': '3', 'unit': None}, '3'),
+                                       '__number__2': ({'value': '2', 'unit': None}, 'two')}
+            >>> self._tag_number_in_text(text)
+            i want to buy __number__1 apples and more than __number__2 bananas
```

Comment:
Minor, example output is wrong as it sorts the detected values in descending order of length of corresponding original_texts

Reply:
Fixed